app/model/editions/client/ClientArticleMetadata.scala

```diff
@@ -0,0 +1,99 @@
+package model.editions.client
+
+import model.editions.{ArticleMetadata, Image, ImageOption}
+import play.api.libs.json.Json
+
+case class ClientArticleMetadata (
```

Comment:
Huh -- how did we get away with not modelling this before? Did we just transparently pass the overrides through as JSON?

Reply:
Looks like we have some models [here](https://github.com/guardian/facia-scala-client/blob/master/facia-json/src/main/scala/com/gu/facia/client/models/Collection.scala#L126) which we could reuse.